### PR TITLE
Remove 'ori' field from Incident

### PR DESCRIPTION
**Pull request by AlexNisnevich**
Merge #11 first for migration support.

### CODE REVIEW

File: db/migrations/20161019_1230_remove_contracting_ori_field.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Migration to remove the 'ori' field for Incidents and
+# rename the GeneralInfo 'contracting_for_ori' field to 'ori'.
+class RemoveContractingOriField < DynamoDB::Migration::Unit
+  def update
+    logger = Logger.new(STDOUT)
+
+    logger.info "Running migration #{self.class.name}:"
```
Comment (wetchler) [inline db/migrations/20161019_1230_remove_contracting_ori_field.rb:7]: Does dynamodb-migration not print anything to stdout by default?

File: db/migrations/20161019_1230_remove_contracting_ori_field.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    incident_ids = Incident.all.map(&:id)
+    logger.info "Updating #{incident_ids.length} Incident records ..."
+    incident_ids.each do |id|
+      client.update_item(
+        table_name: "#{Dynamoid.config.namespace}_incidents",
+        key: { 'id' => id },
+        update_expression: "REMOVE ori"
```
Comment (wetchler) [inline db/migrations/20161019_1230_remove_contracting_ori_field.rb:15]: What would happen if it has no ori? Do we need a condition check here? I'm wondering about the case where a migration fails partway through and we have to re-run it (not totally unlikely)

File: db/migrations/20161019_1230_remove_contracting_ori_field.rb (hunk 1 above)
Comment (AlexNisnevich) [reply db/migrations/20161019_1230_remove_contracting_ori_field.rb:7]: Nope, it does not.

File: db/migrations/20161019_1230_remove_contracting_ori_field.rb (hunk 2 above)
Comment (AlexNisnevich) [reply db/migrations/20161019_1230_remove_contracting_ori_field.rb:15]: Nope, based on my testing, it seems that REMOVE only removes a field if it exists.

**Comment by wetchler** [comment]
lgtm

**Pull request merged**